mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/uwazi/entry/UwaziEntryFragment.kt

```diff
@@ -125,7 +125,7 @@ class UwaziEntryFragment :
 
     private fun initView() {
         with(binding) {
-            this!!.toolbar.backClickListener = { nav().popBackStack() }
+            this!!.toolbar.backClickListener = { onBackPressed() }
```

Comment:
This can reproduce a crash if the view binding is not found.

We should check if the binding id not null and then ask an access for it 
``       
 if (binding == null) return
 with(binding!!)
 {
            toolbar.backClickListener = { onBackPressed() 
}
``